tests/manage/pv_services/test_delete_project_with_pvc_and_pods.py

```diff
@@ -0,0 +1,163 @@
+"""
+The purpose of this test case is to delete at least 2 projects:
+One project with just pvcs and the other with pvcs attached to a pod
+"""
+import logging
+import time
+import pytest
+from itertools import cycle
+
+from ocs_ci.framework.testlib import ManageTest, tier1
+from ocs_ci.ocs import constants, ocp
+from ocs_ci.ocs.cluster import CephCluster
+from ocs_ci.ocs.resources.ocs import OCS
+from tests import helpers
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@pytest.mark.polarion_id("OCS-278")
+@pytest.mark.parametrize(
+    argnames=["pvcs_num"],
+    argvalues=[
+        pytest.param(
+            100, marks=pytest.mark.tier1
+        ),
+        pytest.param(
+            1000, marks=pytest.mark.scale
+        )
+    ]
+)
+class TestDeleteProjectWithPVCAndPods(ManageTest):
+    """
+    Create at least 2 projects, one with just pvcs and the other with pvcs
+    attached to pod, and delete them.
+    """
+
+    @pytest.fixture()
+    def setup(self, storageclass_factory):
+        """
+        Create an OCP+OCS Setup, as well as two Storage classes:
+        one for CephFS and one for RBD.
+        Scale the setup to at-least 800 pvcs/pods.
+        """
+        self.ocp_setup = ocp.OCP()
+        self.ocs_setup = OCS()  # TODO: Scale the setup to at-least 800 pvcs/pods
+        self.ceph_cluster = CephCluster()
+
+        log.info("Creating CephFS Storage class")
+        self.cephfs_sc = storageclass_factory(interface=constants.CEPHFILESYSTEM)
+        log.info("Creating RBD Storage class")
+        self.rbd_sc = storageclass_factory(interface=constants.CEPHBLOCKPOOL)
+
+    def test_delete_project_with_pvc_and_pods(
+        self, project_factory, multi_pvc_factory,
+        pvc_factory, pod_factory, pvcs_num
+    ):
+        # Start with Project 1
+        log.info("Creating Project")
+        project_1 = project_factory()
+
+        log.info("Creating {} CephFS PVCs".format(pvcs_num))
+        # Generate a given number of CephFS PVCs, some RWO and some RWX
+        rwo_rwx = cycle([constants.ACCESS_MODE_RWO, constants.ACCESS_MODE_RWX])
+        cephfs_pvcs = [pvc_factory(interface=constants.CEPHFILESYSTEM,
```

Comment:
Seems like 1600 pvcs will be created (800*2).
I think

Reply:
Isn't pvcs_num either 100 or 1000 (Depending on the scale)? In that case I fail to see how that'd happen?

Reply:
yes, my mistake